distage/distage-model/src/main/scala/izumi/distage/model/definition/dsl/AbstractBindingDefDSL.scala

```diff
@@ -88,14 +93,59 @@ trait AbstractBindingDefDSL[BindDSL[_], SetDSL[_]] {
     *
     * @see Guice wiki on Multibindings: https://github.com/google/guice/wiki/Multibindings
     */
-//  final protected def setOf[T: Tag](implicit pos: CodePositionMaterializer): SetDSL[T] = {
+  //  final protected def setOf[T: Tag](implicit pos: CodePositionMaterializer): SetDSL[T] = {
   final protected def many[T: Tag](implicit pos: CodePositionMaterializer): SetDSL[T] = {
     val setRef = registered(new SetRef(Bindings.emptySet[T]))
     _setDSL(setRef)
   }
 
-//  @deprecated("use .setOf", "14.03.2019")
-//  final protected def many[T: Tag](implicit pos: CodePositionMaterializer): SetDSL[T] = setOf[T]
+  final protected def bind[T: Tag](implicit pos: CodePositionMaterializer): MultipleDSL[T] = {
+    bindImpl[T](ImplDef.TypeImpl(SafeType.get[T]))
+  }
+
+  final protected def bind[T: Tag](instance: T)(implicit pos: CodePositionMaterializer): MultipleDSL[T] = {
```

Comment:
Could you make rename this to `bindValue` and add `bind[T](instance: => T)` (analogous to current from/fromValue & add/addValue distinction)

Reply:
Done.